pkg/shares/namespaced_padding.go

```diff
@@ -0,0 +1,34 @@
+package shares
+
+import (
+	"bytes"
+
+	"github.com/celestiaorg/celestia-app/pkg/appconsts"
+	"github.com/celestiaorg/nmt/namespace"
+)
+
+// NamespacedPaddedShare returns a share that acts as padding. Namespaced
+// padding shares follow a blob so that the next blob may start at an index that
+// conforms to non-interactive default rules.
+func NamespacedPaddedShare(ns namespace.ID) Share {
```

Comment:
Is the namespace here always the same as the blob before it?

Reply:
Yes. I should make that explicit in the godoc. How about?

```go
// NamespacedPaddedShare returns a share that acts as padding. Namespaced
// padding shares follow a blob so that the next blob may start at an index that
// conforms to non-interactive default rules. The ns parameter provided should
// be the namespace of the blob that precedes this padding in the data square.
```

Reply:
Resolved by https://github.com/celestiaorg/celestia-app/pull/1246/commits/ef002ec4cf2869c3c061e07ae24c689193f9a2ec